src/app/workflow/version-modal/version-modal.component.html

```diff
@@ -35,17 +35,17 @@ <h4 mat-dialog-title>{{ isPublic || !canWrite ? 'View' : 'Edit' }} Version Tag</
         </div>
         <div class="form-group form-group-sm">
           <label class="col-sm-3 col-md-3 col-lg-3 control-label">
-            Workflow Path:
+            {{ (isService$ | async) ? 'Service Path:' : 'Workflow Path:' }}
           </label>
           <div class="col-sm-9 col-md-9 col-lg-9">
             <input
               type="text"
               class="form-control"
-              name="workflow_path"
+              name="{{ (isService$ | async) ? 'service_path' : 'workflow_path' }}"
               [(ngModel)]="version.workflow_path"
               minlength="3"
               maxlength="128"
-              [pattern]="validationPatterns.workflowDescriptorPath"
+              [pattern]="(isService$ | async) ? validationPatterns.serviceDescriptorPath : validationPatterns.workflowDescriptorPath"
               required
               [matTooltip]="tooltip.workflowPath"
               placeholder="e.g. CancerCollaboratory/dockstore-tool-liftover"
```

Comment:
This comment for line 52, which I can comment on because you didn't change it. Following up on my comment on the screenshot, I think you should just not have the input at all if a service, and display plain text instead. A disabled input implies there's some way of enabling it.

Reply:
Changed the input to text